content/graphing/infrastructure/containermap.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Container Map Overview
+kind: documentation
+aliases:
+  - /infrastructure/hostmap/
+  - /guides/hostmap
+further_reading:
+- link: "graphing/infrastructure/livecontainers"
+  tag: "Graphing"
+  text: Get real-time visibility of all of the containers across your environment
+- link: "graphing/infrastructure/process"
+  tag: "Graphing"
+  text: Understand what is going on at any level of your system
+---
+
+## Overview
+
+Like [Host Maps][1], [Containers Maps][2] give you a big picture of the health of your containers. Datadog integrates with ECS, Docker, and Kubernetes. You can see all of your containers together on one screen with customized groupings and filters, as well as metrics made instantly comprehensible via color and shape. 
```

Comment:
Do we want "ECS, Docker, Kubernetes, and more"?

Reply:
probs, will amend